src/pages/components/dropdown/usage.mdx

```diff
@@ -617,15 +620,19 @@ Use when you can select only one option from a list at a time.
 
 ## Multiselect
 
-Use when you can select multiple options from a list or to filter information.
+Use a multiselect when the user needs to select multiple options from a list of
+predefined options. Multiselects are a good option when the user needs to filter
+or sort content on a page based on multiple criteria. A checkbox icon is
+included for each option in the list to enable multiselection.
 
 #### Making a selection
 
 - By default, the dropdown displays placeholder text in the field when closed.
```

Comment:
This sorta reads like there is always placeholder text, which has already been listed above as optional. How about...
```suggestion
- By default, the dropdown displays any placeholder text in the field when closed.
```